.node-version

```diff
@@ -1 +1 @@
-14.17
\ No newline at end of file
+14.20
```

Comment:
Angular v15 suports Node v14.20 or higher. I also updated `"engines"` in `package.json` to be the same as in the Angular repo.